synapse/config/consent_config.py

```diff
@@ -54,6 +62,8 @@
 #   block_events_error: >-
 #     To continue using this homeserver you must review and agree to the
 #     terms and conditions at %(consent_uri)s
+#   require_at_registration: False
+#   policy_name: Privacy Policy
```

Comment:
if this is only used for registration, can it be called `registration_policy_name` ?

Reply:
I'd rather not, as this will eventually be used for other things if https://github.com/matrix-org/matrix-doc/pull/1692 goes through.